app/controllers/api/base_controller.rb

```diff
@@ -33,7 +34,11 @@ def pundit_user
   end
 
   def authorization
-    Authorization.new(prx_auth_token) if prx_auth_token
+    if prx_auth_token
+      Authorization.new(prx_auth_token)
+    elsif api_admin_token?
+      Authorization.new(nil, true)
```

Comment:
Kinda hacky - but added a new flavor of `Authorization` when you have one of those API admin tokens (like DTR does).

This grants you read-only access to the authorized API endpoints.  Though if you try to edit anything there, it's going to 500, since Pundit will assume you have a `prx_auth_token`.  (I'll open a ticket after this, to address that - but I don't really want to give these API admins write access anyways).